### PR TITLE
Add SIG Advocacy and Outreach with the permit of TOC

### LINKED ISSUE #291: [Proposal] Setup a new SIG for Advocacy and Outreach
# Background
Community growth is very important for any community. Advocacy and outreach can dramatically help with that. And the key point is that it should not be done by one person or privately.

As far as I can see, there's a couple of forks are interested in this area.

# Why
As an open-source community, we should discuss everything as possible as we can. Discuss advocacy and outreach in an open way is reasonable. Setup this kind of SIG can gather much more great ideas and innovation.

# Benefit
* Less private conversations
* A new channel for those who might want to join us
* As a possible bridge between the software engineers, community managers, and marketing
* Less unnecessary disturb to those who are not interested in this

# Who
Basically, anyone who is interested in the discussion around advocacy and outreach. No matter you're a software engineer, technical writer, or marketing, community growth needs you.

## Potential members
I guess these people might be the potential initial members:
* @shaowenchen 
* @FeynmanZhou 
* @JiannaLiu
* @LinuxSuRen 

# Concern
* There are not enough people to join this SIG.
  * 3 people should be enough to startup it

# Reference
https://www.jenkins.io/sigs/advocacy-and-outreach/

### PR DESCRIPTION
**Pull request by LinuxSuRen**
close #291

Hi, @shenhonglei @webup @faweizhao26 @FeynmanZhou @shaowenchen, so glad to work with all of you guys. I'm not sure if I write your names correctly. Please leave your comments if it's not your preferred one.

I'd like to vote @FeynmanZhou as the leader of this SIG. Hopefully, it does not bother you.

# Actions
* Promote the message of SIG founded
  * [ ] WeChat
  * [ ] Twitter
  * [ ] Linkedin
  * [ ] Slcak
* Call for the first meeting after the Spring festival

### CODE REVIEW

File: sig-advocacy-and-outreach/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+and attract new people to join KubeSphere community.
+
+## Members
+
+- Pengfei Zhou ([@FeynmanZhou](https://github.com/FeynmanZhou)), Lead
+- Shaowen Chen ([@shaowenchen](https://github.com/shaowenchen)), Member
+- Honglei ([@shenhonglei](https://github.com/shenhonglei/)), Member
+- webup ([@webup](https://github.com/webup/)), Member
+- faweizhao26 ([@faweizhao26](https://github.com/faweizhao26/)), Member
+- Rick ([@linuxsuren](https://github.com/linuxSuRen/)), Member
```
Comment (FeynmanZhou) [inline sig-advocacy-and-outreach/README.md:14]: I would recommend Rick to be a lead in this SIG, SIG-Advocacy-and-Outreach was proposed by Rick : )

Comment (LinuxSuRen) [reply sig-advocacy-and-outreach/README.md:14]: Pengfei has done a lot of community management work before. Of course, I'd be happy to be the co-leader of this SIG.

Comment (FeynmanZhou) [reply sig-advocacy-and-outreach/README.md:14]: Please also add yourself, thanks.

Comment (LinuxSuRen) [reply sig-advocacy-and-outreach/README.md]: Done

**Review by FeynmanZhou — APPROVED** [review]

**Comment by FeynmanZhou** [comment]
cc @rayzhou2017

**Comment by LinuxSuRen** [comment]
hi @FeynmanZhou @rayzhou2017 considering we will put the GSoC 2021 under this SIG. We need to push this forward before the application deadline `February 19 19:00 UTC`

**Review by rayzhou2017 — APPROVED** [review]
/lgtm

**Pull request merged**